### DISCUSSION TITLE
cycling through suggestions with alt + [ does not work on Italian keyboard

### DISCUSSION THREAD

**Discussion by alcarrie** [community/community#14414] [Copilot Conversations]
Something I noticed that is really bothering me is the inability to switch between suggestions when using my Italian keyboard. 
The layout of this keyboard allows the user to type the [ only by pressing alt gr + è first, hence there is no way to cycle through the suggestions as I would have to press alt + alt gr + è key. 

Is there any other way to see the other suggestions when using a keyboard with a different layout? 

![image](https://user-images.githubusercontent.com/67009322/162026488-577d7bb6-c15f-4222-ae9f-4b92824e1e5c.png)

**Comment by k3b** [comment]
alternative for "Show next inline suggestion: Alt + ]" for non english keyboards ?

Same issue here:

I am using "Github Copilot" with android-studio and with vscode on my windows-10 pc with a german keyboard that does not allow to press "Alt + ]"

(On a german keyboard you get "]" via "Alt-GR"+9)

Is there an alternative keyboard-shortcut to reach "Alt + ]" and "Alt + [" ?

**Comment by k3b** [comment]
I found a partial answer at #6993

found via https://github.com/github-community/community/discussions?discussions_q=Alt+%2B+%5D+keyboard&page=2

See also 

* #16382
* #15121
* #7212
